recipes/metamorpheus/build.sh

```diff
@@ -3,13 +3,13 @@ set -eox pipefail
 
 PREFIX=$(echo "${PREFIX}" | tr '\\' '/')
 
-mkdir -p $PREFIX/bin/tools
+mkdir -p $PREFIX/bin $PREFIX/dotnet/tools
 cp -r $SRC_DIR/* $PREFIX/bin
 
 if [[ "${build_platform}" == "win-64" ]]; then
     mv $PREFIX/bin/CMD.exe $PREFIX/bin/metamorpheus.exe
```

Comment:
Does this actually work? I thought the exe might expect to find it's associated dlls alongside it.